hs_core/templates/search/search.html

```diff
@@ -551,6 +894,9 @@ <h4 class="panel-title">
                     var textSearch = $("#id_q").val();
                     var updateURL = "/search/?q=" + textSearch;
                     updateURL += buildURLOnCheckboxes();
+                    if(window.location.hash){
```

Comment:
@xuezhaokun add a space before ```{``` for style consistency here as well as a couple of places with the same if condition above.